internal/transactions/sign.go

```diff
@@ -54,10 +56,26 @@ func sign(
 	services *services.Services,
 	state *flowkit.State,
 ) (command.Result, error) {
-	filename := args[0]
-	payload, err := readerWriter.ReadFile(filename)
+	var payload []byte
+	var err error
+	var filenameOrUrl string
+
+	if signFlags.FromRemoteUrl != "" {
```

Comment:
validate that both are not passed.

Reply:
Yes that's better, but what I mean is that I don't see any validation for using the command like: `flow transaction sign ./tx.rlp --from-remote-url HTTP://test.com` so testing both are not passed

Reply:
update, tested and pushed.